Tools/DumpRenderTree/mac/Configurations/DumpRenderTreeLibrary.xcconfig

```diff
@@ -28,7 +28,7 @@
 PRODUCT_NAME = DumpRenderTree;
 SKIP_INSTALL = YES;
 
-OTHER_LDFLAGS = ;
+OTHER_LDFLAGS = $(WK_COMMON_OTHER_LDFLAGS);
```

Comment:
What did this do? I don't see a OTHER_LDFLAGS definition in Base or BaseTarget, so I would think it's safe to delete this altogether.

Reply:
Good point.  I'll just delete it.

Reply:
(BTW, I think the original intent was simply to delete all the OTHER_LDFAGS for this target.)